testsuite/features/init_clients/proxy_branch_network.feature

```diff
@@ -4,6 +4,8 @@
 # The scenarios in this feature are skipped if there is no proxy
 # ($proxy is nil) or if there is no private network ($private_net is nil)
 
+@skip_if_container_server
```

Comment:
Skipping this is only temporary until we have the RBS tests ready, right?

Reply:
Right, and it might be better if I add a TODO comment there too. 👍🏼

Reply:
Thank you!